packages/application-components/src/components/page-unauthorized/page-unauthorized.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { renderComponent } from '../../test-utils';
+import PageUnauthorized from './page-unauthorized';
+
+describe('rendering', () => {
+  it('should render help desk link', () => {
+    const rendered = renderComponent(<PageUnauthorized />);
+    expect(rendered.getByText('Help Desk')).toBeInTheDocument();
+  });
```

Comment:
Should we add a test to see that we at least render the title? Otherwise I might get away with changing a bit too much without the test failing.

Reply:
yep, sounds like a good idea